usaspending_api/etl/management/commands/reveal_last_period_award_updates.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from usaspending_api.broker.helpers.last_load_date import get_last_load_date, update_last_load_date
+from usaspending_api.etl.management.helpers.recent_periods import retrieve_recent_periods
+
+logger = logging.getLogger("script")
+
+REVEAL_AWARD_SQL = """
+UPDATE awards
+SET update_date = now()
+WHERE id IN (
+    SELECT DISTINCT faba.award_id
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc ON
+        defc.group_name = 'covid_19'
+        AND faba.disaster_emergency_fund_code = defc.code
+    INNER JOIN submission_attributes sa ON
+        faba.submission_id = sa.submission_id
+    INNER JOIN dabs_submission_window_schedule dabs ON
+        dabs.id = sa.submission_window_id
+        AND dabs.submission_fiscal_year = {year}
+        AND dabs.submission_fiscal_month = {month}
+        AND dabs.is_quarter = {is_quarter}
+    INNER JOIN awards a on faba.award_id = a.id
+);
+"""
+
+
+class Command(BaseCommand):
+
+    """
+    NOTE: This command is necessary to update award values in Elasticsearch that were modified in the
+    latest submission window when that window's submissions are revealed. Elasticsearch calculations for
+    an award's Covid values do not include data from a submission until its submission window's reveal
+    date has been passed. Because incremental loads in Elasticsearch compare the award's update_date field
+    to a rolling date when determining whether it should be recalculated, the data will not automatically
```

Comment:
I think the sentence from LN40 - 42 could be reworded, not sure what the "rolling date" is and while I understand the gist I'm not sure most people reading this will understand what it being discussed

Reply:
How about this? `Because an incremental load in Elasticsearch uses the award's update_date to determine the delta since it last ran, the award data will not be refreshed after a submission closes until it's update_date is touched`

Reply:
Sounds great